### DISCUSSION TITLE
How to run examples of QNEthernet with Arrduino IDE

### DISCUSSION THREAD

**Discussion by igor-sadalski** [ssilverman/QNEthernet#17] [Q&A] [answered]
I git clone the repository into my arduino repository and try to run the most basic example but i get following error, how to fix this?
```
/home/.../QNEthernet/examples/ServerWithListeners/ServerWithListeners.ino:28:24: fatal error: QNEthernet.h: No such file or directory
compilation terminated.
Error compiling for board Teensy 4.1.
```

**Comment by ssilverman** [comment, marked as answer]
It needs to be put inside Arduino’s ‘libraries’ folder. See the “Manual Installation” section here: https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries

I.e. you’d have a libraries/QNEthernet folder.

**Reply by igor-sadalski** [reply]
Great thanks it works perfectly!
